packages/@uppy/dashboard/src/style.scss

```diff
@@ -198,6 +198,13 @@
   fill: #9f9f9f;
 }
 
+// Disallow clicking on all interactive elements
+.uppy-Dashboard--isDisabled .uppy-Dashboard-innerWrap {
+  button, a, input, select, textarea {
```

Comment:
why only button, a, input, select, textarea ? can't we just disable pointer-events for **everything**, e.g the whole dashboard or all children?

alternatively if we want to make it more fine-grained (e.g. still allow some things to be clicked), then I think it's cleaner to do it in javascript (e.g. `if (disabled) return`) so that it won't have unintended consequences

Reply:
Thought it’s more performant to target interactive elements, those that [support the `disabled` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled), than doing `.uppy-Dashboard--isDisabled .uppy-Dashboard-innerWrap *`.

Reply:
what about `.uppy-Dashboard--isDisabled .uppy-Dashboard-innerWrap`?

Reply:
That is currently the case https://github.com/transloadit/uppy/blob/3e796eefe9a3733a114ee2564c238b609032991f/packages/%40uppy/dashboard/src/style.scss#L176-L195, doesn't propagate to children.

![image](https://user-images.githubusercontent.com/1199054/215914497-f321520f-ea50-4aaa-85ee-9194cfad9bc9.png)

Reply:
ah ok! The only problem with doing this selective disabling as I see it is that if someone were to add a different button type, e.g. `<div role="button">` in the future, then this disabling functionality would stop working and nobody will notice.